app/models/post.rb

```diff
@@ -1,3 +1,7 @@
 class Post < ApplicationRecord
-  belongs_to :author_id
+  belongs_to :user
+
+  scope :last_user_posts, ->(user_id) { where(user_id:).last(3) }
+  scope :increace_comments_count, ->(post_id) { find_by(id: post_id).increment!(:comments_counter) }
+  scope :increace_likes_count, ->(post_id) { find_by(id: post_id).increment!(:likes_counter) }
```

Comment:
- [ ] I can't see any method implemented to increase post counter so please implement thus method in post model 💯🙏🏻 as required and please keep each method in its model to keep your functionality as single principle responsiblilty so each file responsible for its methods 🙏🏻💯🔥

 ![image](https://user-images.githubusercontent.com/58553711/220619262-b79d6289-0963-4a89-a32c-661bfdca6484.png)


-  Hint: I would highly recommend to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private. 🙏🏻💯